packages/@aws-cdk/cloud-assembly-schema/lib/artifact-schema.ts

```diff
@@ -84,7 +84,27 @@ export interface TreeArtifactProperties {
   readonly file: string;
 }
 
+/**
+ * Artifact properties for embedded Cloud Assemblies
+ */
+export interface EmbeddedCloudAssemblyProperties  {
```

Comment:
Can you describe in the README of this packge what an `EmbeddedCloudAssembly` - conceptually (what it represents, why it exists) and practically (its relationship with the other primitives in this package).